src/table/utils/get-view-state.ts

```diff
@@ -6,9 +6,9 @@ import { TableLayout, ViewService } from '../../types';
 
 const getViewState = (layout: TableLayout, viewService: ViewService, rootElement: HTMLElement) => {
     if (viewService.viewState && !viewService.viewState.isMultiPage) return viewService.viewState;
-  
+    let totalsPosition;
```

Comment:
Can't you just do set the totalsPosition here and use that for both cases?
```suggestion
    const totalsPosition = viewService.viewState?.isMultiPage ? totalsPosition : undefined
```

Reply:
Why does it have to be undefined when it's not a multipage print?

Reply:
The value is never used by the table itself, so could just as well have it there at all times.

Reply:
Is it just me that is confussed by `viewState` and what is something more of a "rendering option" versus what is a value that some other party will consume? There seem to be no indication of that in this "API"? Or maybe I have completly miss-understood the viewState..

Reply:
I tried to avoid update unit tests since there are unit tests to check if getTotalPosition should be run or not.

Reply:
> Is it just me that is confussed by `viewState` and what is something more of a "rendering option" versus what is a value that some other party will consume? There seem to be no indication of that in this "API"? Or maybe I have completly miss-understood the viewState..

For multi-page pdf, viewstate is used to comunicate between sn-table and download service. It is more complicated than any other scenarios which use viewstate before.

Reply:
> The value is never used by the table itself, so could just as well have it there at all times.

OK, totalsPosition is always there.